libs/utils/sidekick.js

```diff
@@ -36,4 +38,4 @@ export default function init({ createTag, loadBlock, loadScript, loadStyle }) {
 
   // Color code publish button
   stylePublish(sk);
-}
+}
```

Comment:
Trailing newline also missing here
```suggestion
}

```